src/Sarif/ExtensionMethods.cs

```diff
@@ -14,6 +14,23 @@ namespace Microsoft.CodeAnalysis.Sarif
 {
     public static class ExtensionMethods
     {
+        public static IDictionary<string, MultiformatMessageString> ConvertToMultiformatMessageStringsDictionary(this IDictionary<string, string> v1MessageStringsDictionary)
+        {
+            if (v1MessageStringsDictionary == null)
+            {
+                return null;
+            }
+
+            var converted = new Dictionary<string, MultiformatMessageString>();
+
+            foreach (KeyValuePair<string, string> keyValuePair in v1MessageStringsDictionary)
+            {
+                converted[keyValuePair.Key] = new MultiformatMessageString { Text = keyValuePair.Value };
+            }
+
+            return converted;
```

Comment:
>return [](start = 12, length = 6)

The whole method is just:
```
    return v1MessageStringsDictionary?.
```
This works even if the argument is null (I tried it).